docs/api.md

```diff
@@ -1227,6 +1227,20 @@ ServiceOverrides
 <p>Tolerations allow you to schedule pods onto nodes with matching taints.</p>
 </td>
 </tr>
+<tr>
+<td>
+<code>topologySpreadConstraints</code></br>
```

Comment:
The way we generate documentation changed in https://github.com/planetscale/vitess-operator/pull/625. We no longer use `</br>` as it can break in Markdown. Do you mind merging `main` onto your branch and re-generating the docs?